docs/source/config_update.md

```diff
@@ -9,7 +9,7 @@ Like many complex systems, Hyperledger Fabric networks are comprised of both **s
 * **Structure**: encompassing users (like admins), organizations, peers, ordering nodes, CAs, smart contracts, and applications.
 * **Process**: the way these structures interact. Most important of these are [Policies](./policies/policies.html), the rules that govern which users can do what, and under what conditions.
 
-Information identifying the structure of blockchain networks and the processes governing how structures interact are contained in **channel configurations**. These configurations are collectively decided upon by the members of application channels, where transactions between peer organizations happen, and the "orderer system channel" managed by ordering organizations, and are contained in blocks that are committed to the ledger of a channel.
+Information identifying the structure of blockchain networks and the processes governing how structures interact are contained in **channel configurations**. These configurations are collectively decided upon by the members of channels and are contained in blocks that are committed to the ledger of a channel. Channel configurations can be built using a tool called `configtxgen`, which uses a `configtx.yaml` file as its input. You can look at a [sample `configtx.yaml` file here](http://github.com/hyperledger/fabric/blob/release-2.0/sampleconfig/configtx.yaml).
```

Comment:
Do we need to put the release in tags so it adjusts to master/release-2.0/release-1.4 etc. when we cherry pick to other branches? Let me find the syntax for that.

Reply:
We could, I guess, but I hadn't planned to push this to any other branches.

Reply:
OK then. Not needed here.